core/config/locales/en.yml

```diff
@@ -51,11 +51,12 @@ en:
         name: Add File
         current: Current File
       search:
-        button_text: Search
+        button_text: Go
```

Comment:
According this website : http://www.w3schools.com/tags/att_input_placeholder.asp
"The placeholder attribute specifies a short hint that describes the expected value of an input field (e.g. a sample value or a short description of the expected format)."

We should write "Text", "Keywords", ... instead of "Search..."